ui/client/containers/admin/CustomProcesses.js

```diff
@@ -40,13 +40,13 @@ class CustomProcesses extends BaseProcesses {
 
   deploy = (process) => () => {
     this.props.actions.toggleConfirmDialog(true, DialogMessages.deploy(process.name), () => {
-      return HttpService.deploy(process.name).then(() => this.reload())
+      return HttpService.deploy(process.name).finally(() => this.reload(true))
```

Comment:
What is this argument of reload? I see in code only non-argument version of this method. Why you don't do something like `this.props.actions.loadProcessState`?

Reply:
Changed